src/lib/rule2E.ts

```diff
@@ -63,7 +63,7 @@ function getValueIfComboboxOrListbox(
   if (
     node instanceof HTMLInputElement &&
     TEXT_INPUT_TYPES.includes(node.type) &&
-    node.hasAttribute('list')
+    (node.hasAttribute('list') || nodeRole === 'combobox')
```

Comment:
Does this only apply on `<input role="combobox">` and not other things with an explicit role of `combobox`?